pkg/project/getproject.go

```diff
@@ -36,11 +42,15 @@ func GetProject(httpClient utils.HTTPClient, conID, projectID string) (*Project,
 	}
 
 	// send request
-	resp, err := httpClient.Do(req)
-	if err != nil {
-		return nil, err
+	resp, httpSecError := sechttp.DispatchHTTPRequest(httpClient, req, conInfo)
+	if httpSecError != nil {
+		return nil, httpSecError
 	}
 	defer resp.Body.Close()
+	if resp.StatusCode == 404 {
```

Comment:
rather than 404 you could also use the constants from the http package eg  http.StatusNotFound